wxt_cidrbot/cidrbot.py

```diff
@@ -166,17 +169,49 @@ def webhook_request(self, event):
             if org_id == self.orgID:
                 self.message_event(json_string, event_type, webex_msg_sender)
 
+    def check_message_overflow(self, message, room_id, message_id, pt_id, request_type):
+        split_keyword = 'Repo:'
+
+        if len(message) < 4800:
```

Comment:
best to make this value a constant somewhere, maybe self.MAX_MESSAGE_LENGTH or something

Reply:
Changed to self.webex_message_char_limit